ChatBot/agent/agent.py

```
@@ -0,0 +1,601 @@
+
+    def print_menu():
+        print("\n=== Hindi Learning Agent Debug Menu ===")
+        print("1. Fetch a new word")
+        print("2. Check an answer")
+        print("3. Generate hints for a word")
+        print("4. Summarize session")
+        print("5. Exit")
+        print("=======================================")
+
+    def main():
+        student_id = f"student_{random.randint(1000, 9999)}"
```

Ruff (0.8.2)

Undefined name `random`

(F821)